frontend/src/component/pages/shared/Footer.tsx

```diff
@@ -1,14 +1,49 @@
 import React, { Component } from 'react';
 
 import './Footer.scss';
+import { Update } from '../../../entity/Update';
+import { UIFactory } from '../../UIFactory';
+
+interface State {
+  showChangeLogModal: boolean;
+}
 
 interface Props {
+  uiFactory: UIFactory;
   authorName: string;
   authorPortfolio: string;
   version: string;
+  changeLog?: Array<Update>;
+  newUpdateReleased?: boolean;
+  updateLastSeenChangeLog: () => void;
 }
 
-export class Footer extends Component<Props> {
+export class Footer extends Component<Props, State> {
+  state = {
+    showChangeLogModal: false
+  };
+
+  componentDidUpdate(prevProps: Props) {
+    if (prevProps.newUpdateReleased !== this.props.newUpdateReleased) {
+      this.setState({
```

Comment:
If condition so that unnecessary updates do not happen, and show change log modal only if new update is released (https://github.com/short-d/short/pull/529/files#r389289554).